README.md

```diff
@@ -26,6 +26,27 @@ module "sql-database" {
 }
 ```
 
+Usage (Current Version)
+-----
+
+```hcl
+module "sql-database" {
+  source              = "github.com/foreverXZC/terraform-azurerm-database"
```

Comment:
Remove personal account `foreverXZC` here.

Reply:
Yes I am also thinking about it right now. So do you mean to remove old version part and change source to "Azure/database/azurerm"? I am afraid that it seems ambiguous because new version has not been registered, which means the usage in new readme does not match current version.

Reply:
I think the time gap here is OK. Just remember to publish it after the code is merged.